types/jquery.pjax/tsconfig.json

```diff
@@ -12,6 +12,11 @@
         "typeRoots": [
             "../"
         ],
+        "paths": {
```

Comment:
This should only be done if the package really relies on an older version of jquery.